x-pack/legacy/plugins/apm/cypress/ci/prepare-kibana.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+CYPRESS_DIR="x-pack/legacy/plugins/apm/cypress"
+
+echo "1/3 Install dependencies..."
+# shellcheck disable=SC1091
+source src/dev/ci_setup/setup_env.sh true
+yarn kbn bootstrap
+cp ${CYPRESS_DIR}/ci/kibana.dev.yml config/kibana.dev.yml
+echo 'elasticsearch:' >> config/kibana.dev.yml
```

Comment:
We should probably just add this line to `kibana.dev.yml`

Reply:
I did try but unfortunately the `kibana.yml` don't need that particular entry, otherwise, Kibana fails with:

```
node scripts/kibana --no-base-path --csp.strict=false --optimize.watch=false

 FATAL  Error: [elasticsearch]: expected a plain object value, but found [null] instead.
```